app/helpers/application_helper/button/generic_feature_button.rb

```diff
@@ -7,10 +7,8 @@ def initialize(view_context, view_binding, instance_data, props)
   end
 
   def skip?
-    begin
-      return !@record.send("supports_#{@feature}?")
-    rescue NoMethodError # TODO: remove with deleting AvailabilityMixin module
-      return !@record.is_available?(@feature)
-    end
+    ret = @record.try("supports_#{@feature}?")
+    ret = @record.try(:is_available?, @feature) if ret.nil? # TODO: remove with deleting AvailabilityMixin module
+    !ret
```

Comment:
So if neither `supports_` nor `is_available?` are available at the model, we return `true`, meaning the button should be skipped.

Is that correct?